src/Interpreters/ExternalModelsLoader.cpp

```diff
@@ -34,6 +35,18 @@ std::shared_ptr<const IExternalLoadable> ExternalModelsLoader::create(
                 lifetime
         );
     }
+    else if (type == "tensorflow")
+    {
+        int32_t num_threads = 1;
```

Comment:
It is better to do interpreteration in 1 thread, we does not prefer if external libraries create threads that we cant control.